en/sql/function/json_fn.rst

```diff
@@ -9,6 +9,56 @@ JSON functions
 
 .. contents::
 
+.. _fn-json-intro:
+
+Introduction to JSON functions
+===================================
+
+The functions described in this section perform operations on JSON data.
+They have in common several types of input arguments:
+
+  - *json_doc*: a JSON or string that is parsed as JSON
+  - *val*: a JSON or a value that can be interpreted as one of supported JSON
+    scalar types
+  - *json key*: a string as key name
+  - *json path/pointer*: a string that follows the rules explained in
+    :ref:`json-path` and :ref:`json-pointer`.
+
+.. note::
+
+  UTF8 is expected to be the codeset of JSON functions string arguments.
+  Inputs with different codesets are implicitly converted to UTF8. One
+  consequence is that searching a case insensitive collation string with a
+  codeset other than UTF8 may not provide expected results.
+
+The next table shows the differences between *json_doc* and *val* input
+arguments:
+
++-------------------+-----------------------------+---------------------------+
+| Input type        | *json_doc*                  | *val*                     |
++===================+=============================+===========================+
+| JSON              | Input is unchanged          | Input is unchanged        |
++-------------------+-----------------------------+---------------------------+
+| String            | JSON value is parsed from   | Input is converted to     |
+|                   | input                       | JSON STRING               |
++-------------------+-----------------------------+---------------------------+
+| Short, Integer    | Conversion error            | Input is converted to     |
+|                   |                             | JSON INTEGER              |
++-------------------+-----------------------------+---------------------------+
+| Bigint            | Conversion error            | Input is converted to     |
+|                   |                             | JSON BIGINT               |
++-------------------+-----------------------------+---------------------------+
+| Float, Double,    | Conversion error            | Input is converted to     |
```

Comment:
```suggestion
| Float, Double     | Conversion error            | Input is converted to     |
```

Reply:
there was also numeric, but it was accidentally removed :).